### PR TITLE
fix: misspell on return code

### PR DESCRIPTION
**Pull request by bekhzod91**
@DifferentialOrange When I synced the fork, GitHub automatically closed the old pull request(#277). Without any hassle, I created a new one.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 - Support `fetch_schema` parameter for a connection (#219).
+- Corrected misspelling in the error code returned.
```
Comment (DifferentialOrange) [inline CHANGELOG.md:9]: 1. I think it belongs to the ### Fixed section. (It's a mistake that another entry is outside the section, sorry for the confusing.)
2. Let's better talk about the effect in the changelog message, like [Fixed] "Error code on socket error.".

Comment (bekhzod91) [reply CHANGELOG.md]: Done

Comment (DifferentialOrange) [reply CHANGELOG.md]: Thank you! Can you squash the commits and force-push to the branch?

Comment (bekhzod91) [reply CHANGELOG.md]: Done

**Review by DifferentialOrange — APPROVED** [review]

**Pull request merged**